implementation/src/main/java/io/smallrye/jwt/config/JWTAuthContextInfoProvider.java

```diff
@@ -143,6 +144,13 @@ private static JWTAuthContextInfoProvider create(String publicKey, String public
     @ConfigProperty(name = "mp.jwt.verify.publickey.location", defaultValue = NONE)
     private Optional<String> mpJwtLocation;
 
+    /**
+     * @since 1.2
```

Comment:
@radcortez  I believe `1.1`, `1.2` etc should only be in the MP JWT source, here it should be `3.0.0` as not all the new methods/properties we will be adding will be MP JWT rellated so we should use smallrye-jwt versions in `since` tags ?

Reply:
I thought we were using the MP API version for properties coming from the spec. At least others properties only reference 1.0 and 1.1. So we should probably make it consistent across the board.

Reply:
@radcortez Yeah, it was originally there, but I guess it was not correct. For example, we have several `since 2.1.3` tags now so having `1.0`, `2.1.3`. I propose drop all `1.0`, `1.1`, etc and only refer to smallrye versions, and may be add some plain Java doc comments that a given property is part of MP JWT 1.2, etc.